app.py

```diff
@@ -50,6 +50,9 @@
     ShotgunAudioExporterUI,
     ShotgunHieroObjectBase,
 )
+
+import base_hooks
```

Comment:
Why the absolute import? What if this app runs next another app which also imports base hooks? 

We typically very very rarely do absolute imports and if we do, we name things explicitly (e.g. the `import tk_hiero_export` above is slightly better but still unconventional!)

See the nuke review app for another way to do this: https://github.com/shotgunsoftware/tk-nuke-quickreview/blob/master/app.py#L30